integration-tests/tests/upgrade.rs

```diff
@@ -0,0 +1,50 @@
+#![cfg(test)]
+
+use axelar_gateway::testutils::initialize;
+
+use axelar_gateway::AxelarGatewayClient;
+use soroban_sdk::{testutils::Address as _, Address, Env, String};
+
+mod old_contract {
+    soroban_sdk::contractimport!(
+        file = "../integration-tests/tests/testdata/axelar_gateway_old.wasm"
```

Comment:
instead of checking it in, retrieve the build dynamically from the `target`? you can use cargo macros to get the release type etc. to get the path perhaps. Maybe ask chatgpt/claude for help

Reply:
I tried asking ChatGPT and Claude for help in multiple ways, but none of my attempts worked. I believe the issue is that Stellar SDK procedural macros, like `contractimport!`, require a static string for the file path and don’t support dynamically generated paths via other macros. I'm inclined to keep it as it is for now, but I agree that we should move this under the gateway test.

Reply:
Ok, add comments on how you generated these for reproducibility

Reply:
updated.